### PR TITLE
[FIX] l10n_it_edi: add edi prediction context

### PR DESCRIPTION
**Pull request by sold-odoo**
For the italian localization the predictions for the taxes
and products of each move lines has to be avoided when
dealing with EDI imports.
Thus context should contain this information as to pass
this specification onto the prediction model (account_predictive_bills).

opw-3201391
opw-3172035

### CODE REVIEW

**Comment by sold-odoo** [comment]
This PR is done in tandem with: #38079

File: addons/l10n_it_edi/models/account_edi_format.py [hunk 1]
```diff
@@ -531,7 +531,10 @@ def _import_fattura_pa(self, tree, invoice):
 
             # Setup the context for the Invoice Form
             invoice_ctx = invoice.with_company(company) \
-                                 .with_context(default_move_type=move_type)
+                                 .with_context(
+                                    default_move_type=move_type,
+                                    edi_l10n='it'
```
Comment (jbw-odoo) [inline addons/l10n_it_edi/models/account_edi_format.py:536]: Is this used somewhere ?

**Comment by jbw-odoo** [comment]
Hello. Is this linked to a task or a ticket ?

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by william-andre** [comment]
@robodoo rebase-ff

**Pull request closed without merging**